apps/web/store/bulkBookmarksAction.tsx

```diff
@@ -0,0 +1,44 @@
+// reference article https://refine.dev/blog/zustand-react-state/#build-a-to-do-app-using-zustand
+import { create } from "zustand";
+
+import type { ZBookmark } from "@hoarder/shared/types/bookmarks";
+
+interface BookmarkState {
+  selectedBookmarks: ZBookmark[];
+  isBulkEditEnabled: boolean;
+  handleBulkEdit: (isEnabled: boolean) => void;
+  toggleBookmark: (bookmark: ZBookmark) => void;
+  getSelectedBookmarks: () => ZBookmark[];
+}
+
+const selectedBookmarksStore = create<BookmarkState>((set, get) => ({
```

Comment:
Looking at zustand's documentations, those stores are meant to be `hooks`. So they should start with the `use` prefix, and similarly when you import them the import name should have `use` as a prefix.

Reply:
Fixed, Renamed store name